### PR TITLE
fix(cli): Don't specify --user flag when running in an pip activated virtualenv

### PR DESCRIPTION
**Pull request by tl24**
This fixes an issue when using the python-pip template running in an activated virtualenv.  Issue #705

### CODE REVIEW

File: packages/cdktf-cli/templates/python-pip/.hooks.sscaff.js [hunk 1]
```diff
@@ -32,7 +32,11 @@ exports.post = options => {
   }
 
   writeFileSync('requirements.txt', pypi_cdktf, 'utf-8');
-  execSync('pip3 install --user -r requirements.txt', { stdio: 'inherit' });
+  installArgs = '';
```
Comment (ansgarm) [inline packages/cdktf-cli/templates/python-pip/.hooks.sscaff.js:35]:
```suggestion
  let installArgs = '';
```
Let's keep it a local variable, JavaScript works a bit different than Python here 😅

**Review by ansgarm — CHANGES_REQUESTED** [review]

**Comment by ansgarm** [comment]
Thank you for the PR, @tl24 💯 Looks good besides a small thing 🙂

File: packages/cdktf-cli/templates/python-pip/.hooks.sscaff.js (hunk 1 above)
Comment (tl24) [reply packages/cdktf-cli/templates/python-pip/.hooks.sscaff.js:35]: Sure thing.  Working in too many different languages, hard to remember all the syntax differences...

Comment (ansgarm) [reply packages/cdktf-cli/templates/python-pip/.hooks.sscaff.js]: Yeah, I thought so 😁 Been there as well

**Review by ansgarm — APPROVED** [review]

**Pull request merged**